modules/olm-mirroring-catalog-colocated.adoc

```diff
@@ -35,16 +35,14 @@ $ oc adm catalog mirror \
     <mirror_registry>:<port>/<namespace> \ <2>
     [-a ${REG_CREDS}] \ <3>
     [--insecure] \ <4>
-    [--index-filter-by-os='<platform>/<arch>'] \ <5>
```

Comment:
The `--index-filter-by-os='<platform>/<arch>'` still work, we should NOT remove it. But, the `--index-filter-by-os='.*'` doesn't work. We should remove the `.*` described in https://docs.openshift.com/container-platform/4.10/installing/disconnected_install/installing-mirroring-installation-images.html#olm-mirror-catalog-colocated_installing-mirroring-installation-images, like the screenshot shows:
<img width="477" alt="image" src="https://user-images.githubusercontent.com/15416633/181669595-7b731852-ca8e-44ee-84c0-a54eb48c6728.png">

Reply:
Understood. Addressed. Thanks!